prime-router/src/main/kotlin/cli/main.kt

```diff
@@ -42,10 +41,11 @@ class RouterCli : CliktCommand(
         option("--input_fake", help = "fake the input").int().convert { InputSource.FakeSource(it) },
         option("--input_dir", help = "<dir>").convert { InputSource.DirSource(it) },
     ).single()
-    private val inputSchema by option("--input_schema", help = "<schema_name>").required()
+    private val inputSchema by option("--input_schema", help = "<schema_name>")
 
```

Comment:
Can no longer make this required because the --list option does not require this.